libs/rollup.config.js

```diff
@@ -128,6 +165,26 @@ export default [
     ...browserConfig
   },
 
+  /**
+   * SDK bundled for prebuilt package file to be used in browser
+   * Does not include libs but does include polyfills and all deps/dev deps
+   */
+  {
+    input: 'src/sdk/sdkBrowserDist.ts',
+    output: [
+      {
+        file: pkg.sdkBrowserDistFile,
+        ...browserDistFileOutputConfig
+      },
+      {
+        file: pkg.sdkBrowserDistFileMini,
+        ...browserDistFileOutputConfig,
+        plugins: [terser()]
```

Comment:
Interesting, I didn't know you could add plugins to an output config. But good call on adding the terser! Do we really need both the regular and mini versions of the file?

Reply:
yeah i don't think we really need the regular. ill just go an remove it!